pyaedt/generic/touchstone_parser.py

```diff
@@ -475,3 +477,74 @@ def read_touchstone(file_path):
     """
     data = TouchstoneData(touchstone_file=file_path)
     return data
+
+
+@pyaedt_function_handler()
+def check_touchstone_files(folder="", passivity=True, causality=True):
+    """Check passivity and causality for all touchstone files included in folder.
+
+    Parameters
+    ----------
+    folder : str
+        folder path.
+    passivity : bool, optional
+        Whether the passivity check is enabled, The default is ``True``.
+    causality : bool, optional
+        Whether the causality check is enabled. The default is ``True``.
+    Returns : dict
```

Comment:
Please follow the docstring style

Reply:
What is not correct ?

Reply:
The format, it should be something like:

Returns
 -------
        dict
            ....

Reply:
I just changed that on my last commit.